examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/algorithm_script_parametrized_circuit.py

```diff
@@ -27,18 +27,18 @@
 # Use the device declared in the creation script
 device = AwsDevice(os.environ["AMZN_BRAKET_DEVICE_ARN"])
 
-angle = FreeParameter("angle")
-parametrized_circuit = Circuit().rx(0, angle)
+theta = FreeParameter("theta")
+parametrized_circuit = Circuit().rx(0, theta)
 
 counts_list = []
-angle_list = []
+theta_list = []
 for i in range(5):
-    angle_value = np.pi * np.random.randn()
+    theta_value = np.pi * np.random.randn()
```

Comment:
why is this randn and not rand?

Reply:
It reflects the same code from the non-parametric version of algorithm script. We could change both of them to `rand`. This is a getting started example so I think any choice would make sense.